### PR TITLE
Added basic schema

### PR DESCRIPTION
**Pull request by cohnjesse**
While developing the dependency verification app it was discovered that we need a schema in vets-json-schema for the form that will be submitted, the 21-0538. This PR adds that schema.

### CODE REVIEW

File: src/schemas/21-0538/schema.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+import cloneDeep from 'lodash/cloneDeep';
+import pick from 'lodash/pick';
+import commonDefinitions from '../../common/definitions';
+
+const definitions = cloneDeep(commonDefinitions);
+
+const schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'DEPENDENCY VERIFICAION (0538)',
```
Comment (micahchiang) [inline src/schemas/21-0538/schema.js:9]: Verificaion should be Verification

**Review by micahchiang — COMMENTED** [review]
Not sure how it happened but this PR includes two new schemas in `/dist` instead of just one.

File: src/schemas/21-0538/schema.js (hunk 1 above)
Comment (cohnjesse) [reply src/schemas/21-0538/schema.js]: strange, I removed the extra one

**Review by cbtkc — APPROVED** [review]
Looks good!

**Review by jhonnyoddball — APPROVED** [review]

**Pull request merged**